src/content/url-handling.js

```diff
@@ -54,7 +56,9 @@ export function urlFromState(urlState) {
     case 'calltree':
       query.search = urlState.callTreeSearchString || undefined;
       query.invertCallstack = urlState.invertCallstack ? null : undefined;
-      query.jsOnly = urlState.jsOnly ? null : undefined;
+      query.implementation = urlState.implementation === 'all'
+        ? undefined
```

Comment:
nit: why not `null` ? I remember you told me we try to use `null` instead of `undefined` :)